doc/source/ScriptingAPI/SpecialVariables/index.md

```diff
@@ -0,0 +1,32 @@
+
+# Special Variables #
```

Comment:
I wasn't sure about the best term for these. They're somewhat like reserved variables or keywords, but they're not immutable.